### DISCUSSION TITLE
Mix of two pieces between Alpha and Leipzig

### DISCUSSION THREAD

**Discussion by Dunhill69** [franciscoBSalgueiro/en-croissant#414] [Show and tell]
I've made a mix of two pieces between Alpha and Leipzig, here they are to see if you can add them.

[dunhill.css.zip](https://github.com/user-attachments/files/17351251/dunhill.css.zip)

**Comment by franciscoBSalgueiro** [comment]
I'd prefer not to add mixes of other piece sets, only original designs.
